src/core/CCList.mli

```diff
@@ -1,3 +1,12 @@
+(*
```

Comment:
is this an effect of the promotion, btw?

Reply:
yes. The autogenerated file is promoted from the ``_build`` dir to the ``src`` dir. You may remove it from the source dir and jbuilder will generate it automatically.